server.js

```diff
@@ -9,4 +9,4 @@ app.get("/*", (req, res) => {
     res.sendFile(path.resolve(__dirname, "frontend", "index.html"));
 });
 
-app.listen(process.env.PORT || 5060, () => console.log("Server running..."));
\ No newline at end of file
+app.listen(process.env.PORT || 20131, () => console.log("Server running..."));
```

Comment:
Can you please change this to something more common such as port 3000? Thanks!

Reply:
Sure, no problem. :)